examples/geometry/image_profile.ipynb

```diff
@@ -0,0 +1,128 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 40,
```

Comment:
The cells appear to have been run out of order.
If you are saving a notebook file with outputs, be sure to reset the kernel, clear the outputs, and run all the cells before saving and committing the file.

Reply:
Yes, this sounds better